e2e-tests/functions

```diff
@@ -178,7 +178,7 @@ get_cr() {
 		.spec.users += [{"name":"postgres","password":{"type":"AlphaNumeric"}}] |
 		.spec.users += [{"name":"'${cr_name}'","password":{"type":"AlphaNumeric"}}] |
 		.spec.image = "'$IMAGE_POSTGRESQL'" |
-		.spec.initContainer.image = "perconalab/percona-postgresql-operator:K8SPG-708-12" |
+		.spec.initContainer.image = "'$IMAGE'" |
```

Comment:
@gkech Why do we need to set it explicitly? It should automatically use the main image if the user doesn't provide a custom one, am I right?

Reply:
Yes Slava, this is correct. My main rational for considering setting this is to avoid the logic in the codebase that executes the following API call:

```
func operatorPod(ctx context.Context, cl client.Reader) (*corev1.Pod, error) {
	ns, err := GetOperatorNamespace()
	if err != nil {
		return nil, errors.Wrap(err, "get namespace")
	}

	pod := new(corev1.Pod)
	nn := types.NamespacedName{
		Namespace: ns,
		Name:      os.Getenv("HOSTNAME"),
	}
	if err := cl.Get(ctx, nn, pod); err != nil {
		return nil, err
	}

	return pod, nil
}
```

So instead of making the API call to determine the image, use the specified one and thus, maybe save some exeuction time for the whole e2e suite. If we think that it is not convinient, I can remove it completely. The tests still pass.